### PR TITLE
Apply overscroll to quest screen widgets

### LINKED ISSUE #94: [Bug]: Quest Overview / Tasks Rewards are cut off by nothing
### Bug Description

Screens scroll one text line at a time, which looks great for text but causes multi-line widgets and headers to become "cut off" without there being any visual separation between the visible section and the non-visible section, appearing glitchy

image

### How to Reproduce?

make an overview with many task previews or h1 elements 

### Expected Behavior

haha would it be one of our "bug" reports if we didn't have a solution already loaded (it would not)

https://github.com/terrarium-earth/Hermes/pull/3 demonstrates a method by which the content can take up the whole content area vertically while mid-scroll, preventing the "invisible cutoff" problem, but still show a comfortable margin when at either extreme of the scrolling area (or always if the content fits completely on screen) 

The same principle could be applied to tasks and rewards from there

### Version

1.0.5

### Mod Loader Version

0.20.2

### Mod Loader

_No response_

### Logs or additional context

_No response_

### Code of Conduct

- [X] I have searched the issue tracker and confirmed that the issue has not been reported
- [X] I have checked the FAQ (if one is present) and my issue is not listed
- [X] I have verified that I am using the latest version of the mod
- [X] I have verified that I have installed all the required dependencies for the mod
- [X] I have verified that I do not have Optifine installed. We will close issues if we detect Optifine is in your pack. We cannot fix Optifine issues. Please do not report issues with Optifine present.

### LINKED ISSUE #117: [Feature Request]: widen quest desc text box
### Is your feature request related to a problem?

italics get cut off

### Solution(s)

widening the text box a little so italics on the edge dont get cut off in the description when on the very edge

### Describe alternatives you've considered

_No response_

### Mod Version

1.0.5

### Mod Loader Version

1.20.1

### Mod Loader

Fabric

### Additional context

example: https://imgur.com/a/Zc2SiPA

### PR DESCRIPTION
**Pull request by sisby-folk**
Closes #94
Closes #117 

https://github.com/terrarium-earth/Heracles/assets/55819817/65b4f6f8-b922-48a4-b4fc-ec7f8eefa462

This won't compile - it requires hermes to be published and then bumped here. I just did it locally to test.

Edit: I (ThatGravyBoat) added that it also closes 117 as hermes 1.2.1 also increases the scissor bounds

### CODE REVIEW

**Comment by sisby-folk** [comment]
Rebased and ready for the hermes release now! I've guessed it as being `1.2.1` and added it to my local, but if it gets released as `1.3.0` just let me know or adjust it in properties yourself.

Feel free to review as well.

File: build.gradle.kts [hunk 1]
```diff
@@ -32,6 +32,7 @@ subprojects {
     }
 
     repositories {
+        mavenLocal()
```
Comment (ThatGravyBoat) [inline build.gradle.kts]: It is now published this can be removed

File: forge/build.gradle.kts [hunk 2]
```diff
@@ -14,7 +14,9 @@ dependencies {
 
     forge(group = "net.minecraftforge", name = "forge", version = "$minecraftVersion-$forgeVersion")
 
+    compileOnly("com.teamresourceful:yabn:1.0.3")
```
Comment (ThatGravyBoat) [inline forge/build.gradle.kts]: These should be merged now just pull upstream

**Review by ThatGravyBoat — CHANGES_REQUESTED** [review]

File: forge/build.gradle.kts (hunk 2 above)
Comment (sleepingdragoninn) [reply forge/build.gradle.kts]: done! 2662665

File: build.gradle.kts (hunk 1 above)
Comment (sleepingdragoninn) [reply build.gradle.kts]: sorted! ecff389

**Comment by sisby-folk** [comment]
Ready again

**Review by ThatGravyBoat — APPROVED** [review]

**Pull request merged**